frontend/src/specific_components/file_data.rs

```diff
@@ -120,6 +120,8 @@ pub fn FileData(props: &Props) -> HtmlResult {
 
 #[hook]
 fn use_element_tree(file_id: Id) -> SuspensionResult<UseFutureHandle<Result<ElementTree, String>>> {
+    // TODO use backend::get_file here
```

Comment:
After that we should've factor `file_data.rs` and clean it up so we should not call yewdux from here but we should call it from `backend::get_file` because in the back and folder is the place where we handle the F a statement for the different environment web version, desktop and yewdux ..